src/app.js

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
-import ReactDOM from 'react-dom'
+import ReactDOM from 'react-dom';
+import {StyleRoot} from "radium";
 import MainPage from './components/MainPage';
-import 'normalize.css/normalize.css'
-import './styles/styles.scss'		
 
-ReactDOM.render(<MainPage />, document.getElementById('app'));
+
```

Comment:
To be able to use Media Queries and keyframes with Radium you need to wrap the top level of your app with StyleRoot, thus the changes here.

I also removed normalize and my overall styles SCSS file as they are no longer being used